Umbba-iOS/Umbba-iOS/Global/Resources/Lottie/View/LottieView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  LottieView.swift
+//  Umbba-iOS
+//
+//  Created by 고아라 on 2023/07/13.
+//
+
+import UIKit
+
+import SnapKit
+import Lottie
+
+final class LottieView: UIView {
+    
+    // MARK: - UI Components
+
+    let lottieSplashView: LottieAnimationView = LottieAnimationView(name: "splash_iOS")
```

Comment:
```suggestion
    private let lottieSplashView: LottieAnimationView = LottieAnimationView(name: 리터럴)
```
- 다음 뷰도 private하게 접근하도록 수정할 수 있을 것 같아요
- "splash_iOS"도 리터럴로 빼보면 어떨까요?